internal/config/options.go

```diff
@@ -164,47 +179,50 @@ var defaultOptions = Options{
 		"Strict-Transport-Security": "max-age=31536000; includeSubDomains; preload",
 	},
 	Addr:                    ":443",
-	CertFile:                filepath.Join(fileutil.Getwd(), "cert.pem"),
-	KeyFile:                 filepath.Join(fileutil.Getwd(), "privkey.pem"),
 	ReadHeaderTimeout:       10 * time.Second,
 	ReadTimeout:             30 * time.Second,
 	WriteTimeout:            0, // support streaming by default
 	IdleTimeout:             5 * time.Minute,
 	RefreshCooldown:         5 * time.Minute,
+	GRPCAddr:                ":443",
 	GRPCClientTimeout:       10 * time.Second, // Try to withstand transient service failures for a single request
 	GRPCClientDNSRoundRobin: true,
 }
 
-// NewOptions creates a new Options struct with only viper initialized
-func NewOptions() *Options {
```

Comment:
I don't think the empty Options is used too many places but I feel like we should offer it as a baseline so you can always get a safe Options struct.

Reply:
By empty options you mean this method right?

Reply:
Sorry - yes just a plain NewOptions().  I know there were one or two in the tests.

Reply:
👍